types/rn-randomnumber/rn-randomnumber-tests.ts

```diff
@@ -0,0 +1,7 @@
+import random = require('rn-randomnumber');
+
+random(-100, 25); // $ExpectType number
+
+random(8); // $ExpectError
```

Comment:
```ts
random(8); // $ExpectError
```
that's not true:
```ts
typeof(rn(8))
'number'
```

Reply:
@peterblazejewicz I will fix that.

The documentation and [code](https://cdn.jsdelivr.net/npm/rn-randomnumber@1.0.2/index.js) don't make it clear that it works with just one param. The code relies on `undedined - number`, which then returns a number within `[0, n}`.

Reply:
please, run the code, it works the way I've described. There is no docs for this. I believe this is how rand impl works, give me a rand number (whatever) or let me give a range to pick from.
If you stick with NaN ($ExpectError), the method stamp should be ovverrides:
```ts
declare function randomNoGenerator(min: number, max: number): number;
declare function randomNoGenerator(): number;
```
(give me one from a range, or just give me whatever)

Reply:
@peterblazejewicz as you said `rn(8)` works, so I'll leave it as `declare function randomNoGenerator(min?: number, max?: number): number;`